bin/rebuild

```diff
@@ -31,6 +32,38 @@ versiondb_head() {
   (cd "$VERSIONDB" && git rev-parse HEAD)
 }
 
+save_build_env() {
+  # saving environment information
+  filename="${LSSTSW_BUILD_DIR}/${BUILD}.env"
+  echo "Saving environment information in ${filename}"
+  conda env export | grep -v "^name"> /tmp/lenv
+  echo "environment_name: ${LSST_CONDA_ENV_NAME}" > "${filename}"
+  cat /tmp/lenv | grep "^prefix" >> "${filename}"
+  cat /tmp/lenv | grep -v "^prefix" > /tmp/lenv1
+  if [[ "${LSST_CONDA_ENV_NAME}" != "lsst-scipipe" ]]; then
```

Comment:
Why don't we record the environment for `lsst-scipipe`?

Reply:
The environment name (lsst-scipipe) does not contain any information on the git reference used to create it.
Probably it is not difficult to make this information persistent when the environment is created, maybe just a link. But, in any case, I hope that this use case is dismissed as soon as possible.
Moreover, keeping the nominal behavior in line with what is available now, this should not be required.

Reply:
I'd be inclined to make every conda env, including the default one that bin/deplot creates, use the new naming convention.

Reply:
If we agree on this, no problems from my side, however, we need to change DMTN-110 accordingly.